src/Common/src/Internal/Runtime/ExceptionStringID.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Internal.Runtime
+{
+    /// <summary>
+    /// Represents an ID of a localized exception string.
+    /// </summary>
+    public enum ExceptionStringID
+    {
+        // As a general guideline, try to use the same ID as mscorrc.rc in CoreCLR uses for the same string.
```

Comment:
I am not sure whether it is a good guideline ... the names below are kind of ugly, e.g. `MissingMethod` would look much better than `EeMissingMethod`.

CoreRT is not going to be bug-for-bug compatible with CoreCLR in the type load exception messages.